### PR TITLE
[V2] chore: adding target to generate azure_dd_max_disk_count.go 

### PR DESCRIPTION
**Pull request by landreasyan**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: pkg/azureutils/azure_dd_max_disk_count.go [hunk 1]
```diff
@@ -16,455 +16,721 @@ limitations under the License.
 
 package azureutils
 
-// about how to get all VM size list,
-// refer to https://github.com/kubernetes/kubernetes/issues/77461#issuecomment-492488756
+// Code generated by genertate-vm-to-disk-count-map Makefile target. DO NOT EDIT.
```
Comment (edreed) [inline pkg/azureutils/azure_dd_max_disk_count.go:19]: sp: `generate-vm-to-disk-count-map`

Comment (landreasyan) [reply pkg/azureutils/azure_dd_max_disk_count.go:19]: Grteat 🤦🤣

File: pkg/tool/gen-disk-skus-map.go [hunk 2]
```diff
@@ -45,53 +47,58 @@ var (
...
+
 func init() {
 	klog.InitFlags(nil)
+	flag.Parse()
 }
 
 func main() {
 
 	boilerPlate := `/*
 Copyright 2021 The Kubernetes Authors.
```
Comment (edreed) [inline pkg/tool/gen-disk-skus-map.go:64]: Adding the "generated" comment below causes the `./hack/verify-boilerplate.sh` script to compare the generated file against the `boilerplate.generatego` file which omits the copyright year. You'll need to remove that to get the verify tests to pass.

Comment (landreasyan) [reply pkg/tool/gen-disk-skus-map.go]: Yep, although removing the DO NOT EDIT was enough.

**Pull request merged**